packages/verifiable-credentials/test/create-credentials.test.ts

```diff
@@ -0,0 +1,92 @@
+// https://nodejs.org/api/assert.html
+const assert = require('assert');
+import { Utils } from '@verida/client-ts';
+import Credentials from '../src/credentials';
+import SharingCredential from '../src/sharing-credential';
+import { config, connect } from './config'
+
+
+describe('Credential tests', function () {
+    describe('Credential Units', function () {
+        this.timeout(100000);
+        let encryptedData;
+
+
+        let appContext;
+        let shareCredential;
+        let credential;
+
+        it('Login in user and create app context', async function () {
```

Comment:
This is not something that needs to be tested, as it's not credential related.

You can however put this in a "beforeEach()" so it initializes a new credential instance before each test?